presto-native-execution/src/test/java/com/facebook/presto/nativeworker/PrestoNativeQueryRunnerUtils.java

```diff
@@ -39,14 +40,12 @@
 
 public class PrestoNativeQueryRunnerUtils
 {
-    private static final Logger log = Logger.get(PrestoNativeQueryRunnerUtils.class);
-
-    private static final String DEFAULT_STORAGE_FORMAT = "DWRF";
-
     // The unix domain socket (UDS) used to communicate with the remote function server.
     public static final String REMOTE_FUNCTION_UDS = "remote_function_server.socket";
     public static final String REMOTE_FUNCTION_JSON_SIGNATURES = "remote_function_server.json";
     public static final String REMOTE_FUNCTION_CATALOG_NAME = "remote";
+    private static final Logger log = Logger.get(PrestoNativeQueryRunnerUtils.class);
```

Comment:
Please don't move these if there are no changes.

Reply:
@aditi-pandit These changes were added by the code formatter, but I think it makes sense because the private vars should be after the public ones. I thought it may not be necessary to create a separate commit for this small change, but I can do it if you insist.

Reply:
That's fine. There isn't a need for separate commit.